apps/microsoft-teams/frontend/src/components/config/DisconnectModal/DisconnectModal.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import DisconnectModal from './DisconnectModal';
+import { describe, expect, it, vi } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import { accessSection } from '@constants/configCopy';
+
+describe('DisconnectModal component', () => {
+  it('mounts and renders the correct content', () => {
+    render(<DisconnectModal isShown={true} handleCancel={vi.fn()} handleDisconnect={vi.fn()} />);
+
+    expect(screen.getByText(accessSection.disconnectModal.confirmDisonnect)).toBeTruthy();
```

Comment:
nit: I am pretty new to vitest, but sometimes I find that asserting `toBeTruthy` is just asserting that the node exists, and not that it is visible (like `toBeVisible`). is that the intent here?

Reply:
Good question! I am also still getting comfortable with vitest, but using `toBeTruthy` is something that we used in other apps with vitest to assert that the node exists. If I try to use `toBeVisible` here, it says `Property 'toBeVisible' does not exist on type 'Assertion<HTMLElement>'`. I wonder if there is a a library we can import or a way to set up vitest to have more assertions that we can use. Something to look into in the future!